beacon-chain/rpc/validator_server.go

```diff
@@ -208,3 +208,16 @@ func (vs *ValidatorServer) retrieveActiveValidator(beaconState *pbp2p.BeaconStat
 	}
 	return beaconState.ValidatorRegistry[validatorIdx], nil
 }
+
+func (vs *ValidatorServer) retrieveActiveValidators(beaconState *pbp2p.BeaconState, pubkeys [][]byte) ([][]byte, error) {
+	v := [][]byte{}
+	for _, pk := range pubkeys {
+		_, err := vs.beaconDB.ValidatorIndex(pk)
```

Comment:
Maybe create a db method to return multiple validators ? That way we only have to do one read

Reply:
done